client/layout/style.scss

```diff
@@ -103,6 +103,7 @@
 	background: var( --color-sidebar-background );
 	border-right: 1px solid var( --color-sidebar-border );
 	width: var( --sidebar-width-max );
+	z-index: z-index( 'root', '.is-nav-unification .layout__secondary' );
```

Comment:
The z-index mapping suggests that this is applies to
```
.is-nav-unification .layout__secondary
```

But here, we are applying the style to just
```
.layout_secondary
```

Will this cause issues on pages that don't apply `.is-nav-unification`?

Reply:
Moved the line to `client/assets/stylesheets/_nav-unification.scss`. Felt it was a good place